src/autoscaler/eventgenerator/generator/evaluator.go

```diff
@@ -128,8 +128,8 @@ func (e *Evaluator) doEvaluate(triggerArray []*models.Trigger) {
 		if isBreached {
 			trigger.MetricUnit = appMetricList[0].Unit
 			e.logger.Info("send trigger alarm to scaling engine", lager.Data{"trigger": trigger})
-
 			if appBreaker := e.getBreaker(trigger.AppId); appBreaker != nil {
+				e.logger.Info("circuit-tripped", lager.Data{"appId": trigger.AppId, "consecutiveFailures": appBreaker.ConsecFailures()})
```

Comment:
how about do the logging only when  ConsecFailures() != 0 ?

Reply:
@cdlliuy Updated